performance_manager/lib/postgres_utils.py

```diff
@@ -163,12 +184,15 @@ class MetadataLog(SqlBase):  # pylint: disable=too-few-public-methods
     created_on = sa.Column(sa.TIMESTAMP, server_default=sa.func.now())
 
 
-def unprocessed_files(sql_session: sessionmaker) -> Dict[str, Dict[str, List]]:
+def get_unprocessed_files(
+    path_contains: str, sql_session: sessionmaker
+) -> Dict[str, Dict[str, List]]:
```

Comment:
Gave this function a `path_contains` string to be used by `RT_TRIP_UPDATES` and `RT_VEHICLE_POSITIONS`

Reply:
👍